kotlin-react/src/main/kotlin/react/ReactNode.kt

```diff
@@ -12,6 +12,9 @@ fun ReactNode.asElementOrNull(): ReactElement? = when {
     else -> null
 }
 
+fun ReactNode(string: String): ReactNode =
```

Comment:
`inline`?

Reply:
fixed

Reply:
IDEA does not recommends use inline here
> Expected performance impact from inlining is insignificant. Inlining works best for functions with parameters of functional types
Shell I suppress it? Why do you want to avoid non-inline declaration? Do you want reduce any runtime after compilation?

Reply:
> Shell I suppress it?
Yes

> Do you want reduce any runtime after compilation?
Yes, this method is just for "union emulation" (for compiler only)

Reply:
[Example](https://github.com/JetBrains/kotlin-wrappers/blob/0a23d99440dca7e3e1495a5eab37c87bd0b2348a/kotlin-react/src/main/kotlin/react/TransitionInstance.kt#L1)

Reply:
fixed